modules/custom/az_core/config/quickstart/chosen.settings.yml

```
@@ -4,14 +4,15 @@ disable_search_threshold: 0
 minimum_width: null
 max_shown_results: null
 use_relative_width: false
-jquery_selector: 'select:visible,select#edit-field-az-page-category,select#edit-menu-menu-parent'
+jquery_selector: 'select:visible:not(table select),select#edit-field-az-page-category,select#edit-menu-menu-parent'
```

Currently, it looks like this change also removes Chosen from select elements in card deck forms. I'm not sure if we want that and I'm not sure if any other dropdowns would be affected as well. What would you think about targeting the row weight dropdowns and publication contributor dropdowns more specifically?

```suggestion
jquery_selector: 'select:visible:not(.delta-order select, .field--name-field-az-contributors select),select#edit-field-az-page-category,select#edit-menu-menu-parent'
```